### DISCUSSION TITLE
POWER User can't access the screens

### DISCUSSION THREAD

**Discussion by PGLange** [Budibase/budibase#2000] [Help] [answered]
Hi again,
One last question to complete my app :) (Thank you for the great support 'till now!)
I begun working with roles - the basic role for data input and correction works as it should.

But the Screens that are related to POWER User Role are not accessible. Roles are setup correctly (in BB Platform and the app), but when I log in to the app and navigate the the screen via URL, only the home URL content is shown.
- No Errors in Browsers debug console
It worked once and I don't understand why it is broken now... pls have a look at the following screenshots
BB Platform Usermanagement
![error](https://user-images.githubusercontent.com/86222990/125939357-006939f8-60c6-4471-aa81-bb696f1a7671.PNG)
User in the App
![error2](https://user-images.githubusercontent.com/86222990/125939359-ee9cda81-dea1-4337-82be-3ebaf0985ba0.PNG)
The POWER USER Screen Section
![error3](https://user-images.githubusercontent.com/86222990/125939360-b9796135-cba8-42f7-ba2b-620a935741db.PNG)
And that is displayed under the ?correct? URL
![error4](https://user-images.githubusercontent.com/86222990/125939363-5df7825c-c1be-4d5f-8eef-a980bea9a2ce.PNG)

**Comment by shogunpurple** [comment, marked as answer]
Hi @PGLange so just to clarify, you are trying to access the URL directly from the browser? Or is it a button in your app that navigates to the screen?

**Reply by PGLange** [reply]
hi @shogunpurple 
directly via URL -> I enter the app and then I add /dashboard ....
I'll try to place a link in the basic navigation for testing if a link would solve it ...

**Reply by shogunpurple** [reply]
Ah, I think that may be the issue. If you try adding a link within your app and point it to the `/dashboard` screen and that works, we could have an issue with our routing, which doesn't respect roles correctly if you update the URL manually in the address bar.

**Reply by PGLange** [reply]
@shogunpurple 
Tried the link in NAV, and this works ... Now I'll test IF a basic user can use this link, too ... :)
Take a look at the screenshot, this bug is minor - I think
![error2](https://user-images.githubusercontent.com/86222990/125941199-49450a5f-087a-4691-807f-cca603be180b.PNG)

**Reply by shogunpurple** [reply]
Yep, looks like the client side router hash is not being applied and the server side is trying to render the route. Good catch. Thanks!
